app/src/main/java/de/tum/in/tumcampusapp/services/GeofencingRegistrationService.kt

```diff
@@ -0,0 +1,88 @@
+package de.tum.`in`.tumcampusapp.services
+
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.support.v4.app.JobIntentService
+import android.support.v4.content.ContextCompat
+import com.google.android.gms.location.Geofence
+import com.google.android.gms.location.GeofencingClient
+import com.google.android.gms.location.GeofencingRequest
+import com.google.android.gms.location.LocationServices
+import de.tum.`in`.tumcampusapp.auxiliary.Const.GEOFENCING_SERVICE_JOB_ID
+import de.tum.`in`.tumcampusapp.auxiliary.Utils
+
+
+/**
+ * Service that receives Geofencing requests and registers them.
+ */
+class GeofencingRegistrationService : JobIntentService() {
+
+    private lateinit var mGeofencePendingIntent: PendingIntent
+    private lateinit var locationClient: GeofencingClient
+
+    @SuppressLint("MissingPermission")
+    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
+        super.onStartCommand(intent, flags, startId)
+        return Service.START_STICKY
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        locationClient = LocationServices.getGeofencingClient(baseContext)
+        val intent = Intent(this, GeofencingUpdateReceiver::class.java)
+        mGeofencePendingIntent = PendingIntent.getBroadcast(this, 0,
+                intent, PendingIntent.FLAG_UPDATE_CURRENT);
+        Utils.logwithTag(TAG, "Service started")
```

Comment:
Can you just use `Utils.log()` without the tag? The class name is derived from the stacktrace with that function. Or was there a special reason for using tags?